src/ServiceControl.Transports.AzureStorageQueues/QueueNameSanitizer.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace ServiceControl.Transports.AzureStorageQueues
+{
+    using System;
+    using System.Security.Cryptography;
+    using System.Text;
+    using System.Text.RegularExpressions;
+
+    public class QueueNameSanitizer
```

Comment:
While not going to address a posibility of a disconnect and independent code changes in the source, suggest to add a comment to reference where this class is copied from (ASQ transport).